src/librustdoc/html/static/js/source-script.js

```diff
@@ -211,6 +216,12 @@ const handleSourceHighlight = (function() {
     };
 }());
 
+const addSourceLineAnchorsHref = () => {
+    onEachLazy(document.querySelectorAll(".src-line-numbers a"), e => {
```

Comment:
This is unneeded. If there is an ID, it automatically links to itself if there is no `href`.

Reply:
What is the right clicking supposed to do?

Reply:
Oh indeed. It doesn't propose to copy a link, what a shame... Then I guess it comes to this: is it better to generate `href` through JS or in the HTML directly?

Reply:
Let's see once we have results to think about. ;)